amy/autoemails/actions.py

```diff
@@ -127,6 +127,11 @@ def all_recipients(self) -> str:
         """If available, return string of all recipients."""
         return ""
 
+    def get_merge_data(self) -> Optional[Dict]:
+        """If available return a dict containing per user customizations
+        See https://anymail.readthedocs.io/en/stable/sending/templates/#anymail.message.AnymailMessage.merge_data"""  # noqa
```

Comment:
Recently I found out Flake8 didn't complain about my long line inside `"""` string. You can check if it complains here without `# noqa` comment.

Reply:
@pbanaszkiewicz looks like it does complain without the noqa